client/build.gradle

```diff
@@ -3,7 +3,7 @@ description = ''
 dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version:'2.9.9'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version:'2.10.0'
-    compile group: 'org.apache.httpcomponents', name: 'httpclient', version:'4.5.11'
+    compile group: 'org.apache.httpcomponents', name: 'httpclient', version:'4.5.5'
```

Comment:
Looks like we have degraded httpclient, 4.5.11 is already being used in other Apps, any specific reason?

Reply:
Thanks @AyanDuttaHMCTS  Good Catch .. have updated the version : 5.5.11